packages/drawer/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@lion/drawer",
+  "version": "0.0.1",
```

This should stay to 0.0.0

```suggestion
  "version": "0.0.0",
```

---

fixed